realm/realm-library/src/objectServer/java/io/realm/internal/SyncObjectServerFacade.java

```diff
@@ -190,4 +194,23 @@ public boolean isPartialRealm(RealmConfiguration configuration) {
     public void addSupportForObjectLevelPermissions(RealmConfiguration.Builder builder) {
         builder.addModule(new ObjectPermissionsModule());
     }
+
+    @Override
+    @SuppressFBWarnings("ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD")
+    public void waitForNetworkThreadExecutorToFinish() {
+        // Since the network pool should only consist of remote logout calls at the point where this
```

Comment:
No guarantees as such, but since the test is finishing when this is called, most likely that should be the case (since ending the test also calls `user.logOut()` for all users). 

In any case, no matter what is in it, the result is no longer relevant because the test either already succeeded or failed. I'll reword the description